content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -104,6 +119,9 @@ for p in $(kubectl get pods -l jobgroup=jobexample -o name)
 do
   kubectl logs $p
 done
+```
+
```

Comment:
please add The output is similar to this: as per the guidelines before any output